jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -318,9 +397,170 @@ def vue_update_subset(self, *args):
             if len(self.subset_states) <= index:
                 return
             sub_states = self.subset_states[index]
+
+            # we need to push updates to subset in pixels. to do this when wcs
+            # linked, convert the new input subset parameters from sky to pix
+            wcs = None
+
+            if self.display_sky_coordinates:
+                wcs = self.app._get_wcs_from_subset(sub_states)
+
+                if wcs is not None:
+
+                    if "CircularROI" in self.subset_types[index]:
+                        # convert newly entered sky coords to pixel
+                        xy = [x['value'] for x in sub if 'Center' in x['name']]
+                        rad = [x['value'] for x in sub if 'Radius' in x['name']][0]
+                        pixregion = CircleSkyRegion(center=SkyCoord(*xy*u.deg), radius=rad*u.deg).to_pixel(wcs)
+                        new_xy = (pixregion.center.x, pixregion.center.y)
+                        new_rad = pixregion.radius
+
+                        # convert previous entered sky coords to pixel
+                        orig_xy = [x['orig'] for x in sub if 'Center' in x['name']]
+                        orig_rad = [x['orig'] for x in sub if 'Radius' in x['name']][0]
+                        orig_pixregion = CircleSkyRegion(center=SkyCoord(*orig_xy*u.deg), radius=orig_rad*u.deg).to_pixel(wcs)
+                        new_orig_xy = (orig_pixregion.center.x, orig_pixregion.center.y)
+                        new_orig_rad = orig_pixregion.radius
```

Comment:
did you try getting rid of these and just always updating the internal state?  Does that cause problems?  I just worry that the code-duplication and extra computations will have more cost than just pushing an extra update (with no changes).

Reply:
i haven't tried that yet, maybe this could be a follow up ticket? i'm fairly certain that removing this (and the check for if new matches old to avoid update) will just cause it to update every time, and is less costly than checking every time, but im worried about unintended consequences.

Reply:
hmm, ok, then is it possible to do the comparison in sky (store the orig sky in the dictionary instead of orig pixel and do the "change comparison" based on that)?  It would just be nice to avoid duplicated code and minimize the number of conversions